### PR TITLE
chapter12_reset_password

### PR DESCRIPTION
**Pull request by tenyiting**
chapter12_reset_password

### CODE REVIEW

**Comment by tenyiting** [comment]
ready

**Comment by tenyiting** [comment]
ready

File: app/assets/javascripts/password_resets.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (ghost) [inline app/assets/javascripts/password_resets.coffee]: delete this file please

File: app/assets/stylesheets/password_resets.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the PasswordResets controller here.
```
Comment (ghost) [inline app/assets/stylesheets/password_resets.scss]: delete this file please

File: app/helpers/password_resets_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module PasswordResetsHelper
```
Comment (ghost) [inline app/helpers/password_resets_helper.rb]: delete this file please

File: config/routes.rb [hunk 4]
```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'password_resets/new'
+  get 'password_resets/edit'
```
Comment (ghost) [inline config/routes.rb]: delete these line please

File: test/controllers/password_resets_controller_test.rb [hunk 5]
```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
```
Comment (ghost) [inline test/controllers/password_resets_controller_test.rb]: delete this file please

**Review by ghost — COMMENTED** [review]
cmt

**Comment by tenyiting** [comment]
ready

File: app/mailers/user_mailer.rb [hunk 6]
```diff
@@ -5,17 +5,25 @@ class UserMailer < ApplicationMailer
...
-  def account_activation(user)
-    @user = user
-    mail to: user.email, subject: "Account activation"
-  end
+  #def account_activation
+    #@greeting = "Hi"
+
+    #mail to: "to@example.org"
+    #end
+
```
Comment (ghost) [inline app/mailers/user_mailer.rb]: delete it if you don't use

**Review by ghost — COMMENTED** [review]
small comment

**Comment by tenyiting** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**